### PR TITLE
CI matrix: add openSUSE 15.2 - second attempt

### PR DESCRIPTION
**Pull request by aik-jahoda**
The original CI matrix change was reverted (#60040) because of a failing CI.

part of #57947

original PR: #59574

### CODE REVIEW

**Comment by safern** [comment]
@aik-jahoda did you run a manual runtime build from your branch? It seems like opensuse is missing fonts and the System.Drawing.Tests are failing because of that.

**Comment by aik-jahoda** [comment]
@safern All test are passed and we can merge it. Can you please review?

**Comment by karelz** [comment]
@safern @ericstj can you please review the PR? Then we can merge (and backport to .NET 6.0). Thanks!

**Review by karelz — APPROVED** [review]

**Comment by ericstj** [comment]
From previous review:

> did you run a manual runtime build from your branch?

I looked through the logs from this PR to find where this is running on the queue you added and can't find it:
https://dev.azure.com/dnceng/public/_build/results?buildId=1484879&view=logs&j=71afe768-e493-597c-228c-037834193aeb&t=fe87d331-02fa-5ca6-e382-69b0a11bb0f6

Perhaps you can share the link to the build which tested your changes?

**Comment by aik-jahoda** [comment]
@krwq  I'm little bit lost with build, I have followed your suggestion from https://github.com/dotnet/runtime/pull/60054#issuecomment-936173365 however I can't see any openSUSE tests:
https://dev.azure.com/dnceng/public/_build/results?buildId=1490317&view=results

Could it be caused by `ne(parameters.jobParameters.isSingleFile, true)` when this branch contains change only on one file?

**Comment by safern** [comment]
I don't think we need to touch another file. Where did you see that condition?

The change looks good, I just triggered another pipeline and when done I can help you getting the open suse test results:
https://dnceng.visualstudio.com/public/_build/results?buildId=1494706&view=results

**Comment by aik-jahoda** [comment]
Test results: https://dnceng.visualstudio.com/public/_build/results?buildId=1494706&view=logs&j=b9b7de4d-2126-5581-6453-1576f9aaac5f&t=db048d48-40e8-5399-9b14-d587763f33ff&l=49

There is failing ubuntu arm, however it is unrelated: https://github.com/dotnet/runtime/issues/60825

**Comment by safern** [comment]
Mono runtime tests are also unrelated.

File: eng/pipelines/libraries/helix.yml [hunk 1]
```diff
@@ -36,3 +36,4 @@ steps:
       SYSTEM_ACCESSTOKEN: $(System.AccessToken) # We need to set this env var to publish helix results to Azure Dev Ops
       HelixTargetQueues: ${{ join('+', parameters.helixQueues) }} # Pass queues to MSBuild as env var to avoid need of escaping them
       _Scenarios: ${{ join(',', parameters.scenarios) }} # Pass scenarios to MSBuild as env var to avoid need of escaping comma separated list
+
```
Comment (safern) [inline eng/pipelines/libraries/helix.yml:39]:
```suggestion
```

**Pull request merged**